docs/architecture/microservices/multi-container-microservice-net-applications/implement-api-gateways-with-ocelot.md

```diff
@@ -135,7 +135,7 @@ However, direct-access communication to the microservice, in this case through t
 
 Ocelot is basically a set of middlewares that you can apply in a specific order.
 
-Ocelot is designed to work with ASP.NET Core only. It targets `netstandard2.0` so it can be used anywhere .NET Standard 2.0 is supported, including .NET Core 2.0 runtime and .NET Framework 4.6.1 runtime and up.
+Ocelot is designed to work with ASP.NET Core only. It targets .NET Core App 3.1 in version 14.0 and later, so it can only be used in ASP.NET Core applications. It's not suitable for .NET Framework applications.
```

Comment:
It contradicts with their official statement: https://github.com/ThreeMammals/Ocelot#how-to-install ?

Reply:
Their official statement is outdated... since [13.8.0 (10/28/2019)](https://www.nuget.org/packages/Ocelot/13.8.0) 😅

![image](https://user-images.githubusercontent.com/7357806/75765496-85f17600-5d37-11ea-8b7c-9d7eb9537891.png)

https://www.nuget.org/packages/Ocelot/14.0.11

Reply:
Interesting. I made some suggestion - I don't think we need to explicitly call out their version numbers.

Reply:
ok.